src/components/WebexActivity/WebexActivity.stories.js

```diff
@@ -90,3 +90,29 @@ export const MultiLine = Template.bind({});
 MultiLine.args = {
   activityID: 'activity3',
 };
+
+export const AdaptiveCard = Template.bind({});
+AdaptiveCard.args = {
+  activityID: 'activity9',
+};
+// Modify time data for actions on the fly
+AdaptiveCard.parameters = {
+  mockData: {
+    activities: {
+      ...activities,
```

Comment:
for a future PR, I see all stories in this file use the same activities, creating a new object each time.

However, the activities themselves (each activity object) remain with the same reference. That means that if one activity is changed in a story (eg, adding an action), all other stories will se the updated activity. Not a big problem, but we should, at some point, create a new object for each activity, for example using JSON.parse(JSON.stringify(activities)).